include/mpsl/mpsl_cx_config_bluetooth.h

```diff
@@ -18,7 +18,7 @@
 
 #include <stdint.h>
 
-/** @brief Configures the Bluetooth Radio Coexistence interface.
+/** @brief Configures the Bluetooth Radio Three Wire Coexistence interface.
```

Comment:
This change seems unintentional

Reply:
I think it is intentional. Do you see any issues with it?

Reply:
Sorry, I see this is now docs for the three wire implementation